deployment/v3/testrig/apitestrig/install.sh

```
@@ -88,29 +88,80 @@ function installing_apitestrig() {
+    NFS_OPTION=''
+    S3_OPTION=''
+    config_complete=false  # flag to check if S3 or NFS is configured
+    while [ "$config_complete" = false ]; do
+      read -p "Do you have S3 details for storing apitestrig reports? (Y/n) : " ans
+      if [[ "$ans" == "y" || "$ans" == "Y" ]]; then
+        read -p "Please provide S3 host: " s3_host
+        if [[ -z $s3_host ]]; then
+          echo "S3 host not provided; EXITING;"
+          exit 1;
+        fi
+        read -p "Please provide S3 region: " s3_region
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)